definitions.md

```diff
@@ -233,21 +249,24 @@ In the context of:
 
 #### Ticket
 
-Some unpredictable element generated by the system for two uses:
-
-- as a random challenge to PoSTs.
-- to elect a leader in Expected Consensus.
-
-See more on [Expected Consensus](expected-consensus.md) and [PoSTs](proofs.md).
+A `ticket` is used as a source of randomness in EC leader election. Every block depends on an `ElectionProof` derived from a `ticket`. At least one new `ticket` is produced with every new block. Ticket creation will be described [here](./expected-consensus.md#Ticket-generation).
 
 #### TipSet
 
-A collection of blocks mined by different miners, each an elected leader of a given epoch. All tip sets have the same parent-set and epoch number (height). TODO add picture.
+ A `TipSet` is a set of blocks that have the same parent set and same number of `tickets`, which implies they will have been mined at the same `height`. A `TipSet` can contain multiple blocks if more than one miner successfully mines a block at the same `height` as another miner. `TipSets` contain 1 or more blocks (i.e. null blocks are not included in `TipSets`).
```

Comment:
This is a bit circular Above it says that height is the number of tipsets passed since genesis, and here it is tipset are the blocks with the same height.

Reply:
> i.e. null blocks are not included in `TipSets`

then where are they included? aren't they part of the block technically?

Reply:
Disagree with circularity. The circular bit is just a reframing that helps cement what height is here. Re null blocks, removed. Confusing bit. A Tipset can never include just a null block, they are indeed included as part of the block, technically (through the ticket array).

Null blocks is a confusing concept overall we should get rid of.